### DISCUSSION TITLE
How use the TSH Length and Fipple Factor

### DISCUSSION THREAD

**Discussion by bluesbi** [edwardkort/WWIDesigner#85] [NAF Study] [answered]
Hi again! I'm loving learn and make flutes with the WIDesigner until now this is being the best tool that I used for it!
Today I come up with a question: What does represent the TSH Length? I've learned a way to make the TSH with a man that has a youtube channel called BluebearFlutes and because I have a lot of rivercane right know and because I'm beginning, I decided to work with them as he shows on this video https://www.youtube.com/watch?v=yYLQYhH-xDY. But on the software I notice that the length is to small to represent the entire air path of the SAQ for the splitting edge, soi was supposing that this measures just the rectangle by the south of the flute, close of the splitting edge,  am I correct? and then comes the Fipple factor saying how the bird should cover the path e the rectangle going for close or far of the splitting edge. It is this?
This is the question sorry my bad language.

**Comment by edwardkort** [comment, marked as answer]
The TSH (true sound hole, or sound hole) is the opening from the splitting edge (south) to the flue exit (north). That north-south measurement is its length. If you are using a round TSH, approximate its length and width to get the same area. See Clint Goss's flutopedia (http://www.flutopedia.com/anatomy.htm).

The fipple factor is an empirical value that represents all of the "unmeasurable" effects around the TSH. You should determine this value for your particular flute in the construction/tuning process. See the video that discusses this.
